cps/kobo.py

```diff
@@ -0,0 +1,544 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import base64
+import copy
+import uuid
+import os
+from datetime import datetime, tzinfo, timedelta
```

Comment:
tzinfo and timedelta seem not to be used

Reply:
Done.